BraveRewardsUI/Tipping/TippingView.swift

```diff
@@ -11,6 +12,40 @@ extension TippingViewController {
     
     var gesturalDismissExecuted: (() -> Void)?
     
+    func setInfo(name: String, tipAmount: Double) {
+      let isMonthly = optionSelectionView.isMonthly
+      
+      confirmationView.faviconImageView.image = overviewView.faviconImageView.image
+      confirmationView.subtitleLabel.text = isMonthly ? Strings.TippingMonthlyTitle : Strings.TippingOneTimeTitle
+      
+      confirmationView.infoLabel.text = "\(name)\n\(tipAmount) BAT\(isMonthly ? ", \(Strings.TippingRecurring)" : "")"
+      
+      if isMonthly {
+        confirmationView.monthlyTipLabel.attributedText = {
+          let paragraphStyle = NSMutableParagraphStyle()
+          paragraphStyle.alignment = .center
+          paragraphStyle.lineBreakMode = .byWordWrapping
+          paragraphStyle.lineSpacing = 8.0
+          
+          let text = NSMutableAttributedString(string: "\(Strings.TippingRecurringDetails)\n", attributes: [
+            .font: UIFont.systemFont(ofSize: 12.0, weight: .regular),
+            .foregroundColor: Colors.grey600
+            ])
+          
+          // TODO: Find out where to get the date!
+          text.append(NSAttributedString(string: "6/20/2019", attributes: [
```

Comment:
There's `reconcileStamp` in PublisherInfo that should be used, but I am not sure about logic of it.
I guess after adding a recurring tip, you have to retrieve a publisher again and read its `reconcileStamp`

summoning @kylehickinson for help on this one

Reply:
Hmm I tried this, it always returns 0 for me. So when I calculate the date since 1970, it'll return 1970 lol.

Reply:
@Brandon-T the right time is: `Date(timeIntervalSince1970: TimeInterval(state.ledger.autoContributeProps.reconcileStamp))`

Reply:
Yeah issue was mock ledger not making a real wallet, so no reconcile stamp